doc/Command_Index_gregorio.tex

```diff
@@ -751,9 +751,6 @@ \section{Gregorio Controls}
   \#4 & \TeX\ code & Macros rendering the first syllable; should emit the initial and populate \verb=\gre@opening@syllabletext=.\\
 \end{argtable}
 
-\macroname{\textbackslash GreScoreReference}{\#1}{gregoriotex-main.tex}
```

Comment:
The function is marked as obsolete and will raise an error if used.  No other obsolete or deprecated functions are documented.

If it's supposed to be there (even doing nothing), then it shouldn't be raising an obsolete error.